nupic/research/frameworks/vernon/experiments/meta_cl_experiment.py

```diff
@@ -160,7 +171,7 @@ def create_train_slow_sampler(cls, config, dataset):
     @classmethod
     def create_replay_sampler(cls, config, dataset):
         """Sampler used to augment meta-train testing; "replays" previous classes."""
-        class_indices = cls.compute_class_indices(config, dataset, mode="replay")
+        class_indices = cls.compute_class_indices(config, dataset, mode="all")
```

Comment:
What about that distinction we discussed, of having the replay only use half of the classes? How did you set that?

Reply:
There's a subclassed version of the experiment in `meta_cl_data_split.py` in the project folder that does it.

I figure we should actually use all of the data, so I didn't incorporate that option into the main codebase. We can add it in anytime if we'd like, maybe a three-line change.

Reply:
I think it is good idea to add it. If we want to compare our results to OML for the paper, we should be able to easily reproduce their results within the framework.

Reply:
Fair enough. Can do.

Reply:
There are now two params `slowfast_classes` and `replay_classes` that be passed to the config to list which classes which will be used for which dataset. To replicate OML, one would set `replay_classes=range(0, 481)` and `slowfast_classes=range(481, 963)`.

Reply:
Sounds cool. We should think about the `slowfast` naming in the future, I agree with Subutai that sounds a bit off. Maybe just `train_classes`.